### PR TITLE
Update views.py

### PR DESCRIPTION
**Pull request by preetmodh**
There were some redundant lines of code which I updated efficiently.

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -99,14 +98,11 @@ def post(self, request):
         user = User.objects.get(username=request.user)
         q = request.data.get('type')
 
-        if q == 'Active Post':
```
Comment (iamharshkumar) [inline api/views.py:102]: If the q is Active post then you need to show only active post and vice versa

File: api/views.py [hunk 2]
```diff
@@ -99,14 +98,11 @@ def post(self, request):
...
-            posts = Post.objects.filter(user=user, active=True).order_by('-timestamp')
-            serializer = self.serializer_class(posts, many=True).data
-            return Response(serializer, status=HTTP_200_OK)
-        if q == 'Pending Post':
+        if q == 'Active Post' or q == 'Pending Post':
```
Comment (iamharshkumar) [inline api/views.py]: If the q is Active post then you need to show only active post and vice versa

Comment (preetmodh) [reply api/views.py]: I changed it please review it. now if q if active post it will show active as true otherwise as false.

Comment (iamharshkumar) [reply api/views.py]: Looks good to me 💯 , you can also refactor my other project e.g moviesfy

Comment (preetmodh) [reply api/views.py]: okay sure :)

**Pull request merged**